tests/unit/core/dynamicBadge.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { dynamicBadge } from "@/core/dynamicData";
+
+describe("#dynamicBadge", () => {
+  describe("Url, label, and query", () => {
+    it("displays a badge with a given label, URL and query", () => {
+      expect(
+        dynamicBadge(
```

Comment:
Can we get some test cases using the external URL and the alt text too? The tests are incomplete if only 3 / 5 parameters supported have tests